components/EditForm/index.js

```diff
@@ -76,62 +87,113 @@ export default function EditForm({ project }) {
     }).then((result) => {
       if (result.isConfirmed) {
         deleteProject(id);
-        router.back();
+        router.push("/Active");
```

Comment:
You should use lowercase for all of your routes. Attention! Changing the filenames to lowercase might be a problem with git if you haven't set ignorecase to false (`git config --global core.ignorecase`). In this case, you should rename the folders/files to temporary names (e.g. `Active_temp`), then commit and then change them to lowercase (`active`) and commit again.

Reply:
okay i will fix it